lib/cylc/tests/test_suite_srv_files_mgr.py

```diff
@@ -218,5 +222,24 @@ def test_register(self, mocked_os, mocked_mkdir_p):
                                     str(the_exception))
 
 
+@pytest.mark.parametrize(
+    'version, expected_err',
+    [
+        ('7.9.3', None),
```

Comment:
I see no reason why it should fail with `CYLC_VERSION=7.8.8-35-g892c6`, but why not test it, or something similar?